docs/node-operators/migration-guides/upgrade-mainnet-to-0714.md

```diff
@@ -0,0 +1,663 @@
+
+It is also important to understand the Tendermint configuration parameters as described in the [Tendermint docs ↗](https://docs.tendermint.com/v0.33/tendermint-core/configuration.html)
+
+You are responsible for deciding what parameters you want to use. `vega tm init` generates a config with default values. Values in your config may be changed intentionally. Review and prepare your config carefully.
+:::
+
+
+### 3. Update data node config
+
+:::note Manual process
+We recommend starting it before the upgrade and having configs ready when you need to restart your node. You may do this task asynchronously to another task. This step may be time-consuming, as We introduced many changes since v0.53, and there is no automation - it needs to be done by hand.
+:::
+
+While following the same procedure for a data node as for `vega` and `tendermint` is possible, there are a lot of changes in the data node config, thus we suggest an alternative process:
+
+1. Ensure you have backup first, then remove the data node state: `rm -r <VEGA-NETWORK-HOME>/state/data-node/`
+2. Save a backup of your old data node config: `cp <VEGA-NETWORK-HOME>config/data-node/config.toml <BACKUP-FOLDER>/data-node-config.toml`
+3. Initiate the data node config: `<VEGA-BIN> datanode init --archive --home=<VEGA-NETWORK-HOME> <CHAIN-ID> --force`
+4. Modify generated data node config: `<VEGA-NETWORK-HOME>config/data-node/config.toml` - see section below for important parameters
+
+Use the `--archive` flag to keep all data, forever. Read more about the [data retention modes](../../concepts/vega-chain/data-nodes.md#data-retention-modes) available.
+
+#### Data node config
+Important config keys that you need to update: 
+
+- `AutoInitialiseFromNetworkHistory` - We recommend setting it to `false` when you start the network from scratch (e.g. checkpoint restart), or if there is no other data node available.
+- `ChainID` - Make sure it matches the new chain ID for your network.
+- `Admin.Server.SocketPath` - Path for the Unix `sock` file; Ensure parent directory exists. Example may be `<VEGA-NETWORK-HOME>/run/datanode.sock`.
+- `API.CoreNodeIP` - IP of the server where the Vega core node is running. Often, it is `127.0.0.1.` (localhost).
+- `API.CoreNodeGRPCPort` - Port you expose for the Vega core gRPC node. Default: `3002`.
+- `SQLStore.wipe_on_startup` - Defines if the data node removes data from the postgreSQL after the restart. We recommend setting it to `false`.
+- `SQLStore.UseEmbedded` - If true, internal (managed by the binary itself) postgreSQL is used. We strongly recommend setting it to `false` for production, as it is intended for testing only.
+- `SQLStore.ConnectionConfig` - Update the entire section, as it is where you set PostgreSQL credentials.
+- `NetworkHistory.Enabled` - Enables IPFS network history. See the [network history](#a-bit-about-network-history) section below.
```

Comment:
Do we need to specify that they should update this?  default is true,  I don't see any reason why we would want them to run with it false (?)